relentless/optimize/method.py

```diff
@@ -391,3 +528,88 @@ def line_search(self, value):
         if value is not None and not isinstance(value, LineSearch):
             raise TypeError('If defined, the line search parameter must be a LineSearch object.')
         self._line_search = value
+
+class FixedStepDescent(SteepestDescent):
+    r"""Fixed-step steepest descent algorithm.
+
+    For an :class:`~relentless.optimize.objective.ObjectiveFunction` :math:`f\left(\mathbf{x}\right)`,
```

Comment:
You could apply similar changes to these docs as I suggested to these docs.

Alternatively, you could say that this is a modification of `SteepestDescent` (and refer the user to those docs for details), then just give the equation in lines 558-561 here.